### DISCUSSION TITLE
<textarea prop:value={x}>

### DISCUSSION THREAD

**Discussion by iacore** [solidjs/solid-start#858] [General]
```
// incorrect
      <textarea value={"without prop:"}/>
// correct
      <textarea prop:value={"with prop:"}/>
```

Demo See https://playground.solidjs.com/anonymous/79e5a5f9-6d50-46b5-b388-5a45f19ebd4e

This is documented in https://www.solidjs.com/guides/rendering#attributes-and-props. However, should Solid provide better default?

Also, `prop:value` has syntax error from linting. Don't know why

**Comment by ryansolid** [comment]
Hmm.. yeah it sets the `value` prop if it is dynamic... but gets statically hoisted if not. It's interesting as textarea doesn't have a value attribute. I think you are right, it should skip hoisting.
